tox.ini

```diff
@@ -38,6 +38,11 @@ passenv =
 setenv =
     _PYTEST_TOX_DEFAULT_POSARGS={env:_PYTEST_TOX_POSARGS_DOCTESTING:} {env:_PYTEST_TOX_POSARGS_LSOF:} {env:_PYTEST_TOX_POSARGS_XDIST:}
 
+    # Warn where default encoding is used
+    # https://docs.python.org/3/library/io.html#io-encoding-warning
+    # Testing changes made in https://github.com/pytest-dev/pytest/pull/10328/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449
```

Comment:
I don't really see what the point of this comment is - wouldn't it be better for people to find this via `git blame` if needed?